doc/source/istanbul.md

```diff
@@ -0,0 +1,32 @@
+# 🏘️ Istanbul
+
+Istanbul, even though some still prefer to call it Constantinople, is the name of this city since 1930 when its name was officially changed to Istanbul. The word “Istanbul” is derived from Greek phrase "εἰς τὴν Πόλιν" which means “to the city”. Even though the city was called as “Der Saadet” meaning the “gate to prosperity” in Ottoman Turkish, the name Istanbul was preferred as it was used since 17th century.
+
+Istanbul, a historical city thanks to being the capital of several empires including Byzantium and Ottoman Empire, with its population over 16 million people, is the most populous city of Republic of Turkey (and also of Europe), or with its widely known name, Turkey. As it is the most populous city of Turkey, the city is thought to be the capital of Turkey, but the capital of Turkey is Ankara.
+
+Istanbul is the country’s economic, cultural, and historic center. In fact, Turkey owes 31% of its total GDP to Istanbul. (according to 2019 reports) Moreover, the city’s economy is bigger than that of 130 countries, 25 of which is located in Europe. When it comes to the culture, much of Turkey’s cultural richness had its roots in Istanbul. It is culturally such an important city that in 2010, it was named as European Capital of Culture. If we need to speak about the history of the city, we can not tell it by words. So, we will not discuss its history besides one really and may be the most important event in the city’s history: The Conquest of Istanbul. This was such a huge event that according to some historians, early modern period of history begins with this event. This is because after the fall of Constantinople, the feudalism system in Europe ended.
```

Comment:
Istanbul is the country’s economic, cultural, and historic center. Turkey owes 31% of its total GDP to Istanbul. (according to 2019 reports) Moreover, the city’s economy is bigger than that of 130 countries, 25 of which is located in Europe. When it comes to the culture, much of Turkey’s cultural richness had its roots in Istanbul. It is culturally such an important city that in 2010, it was named as European Capital of Culture. If we need to speak about the history of the city, we can not tell it by words. So, we will not discuss its history besides one really and maybe the most important event in the city’s history: The Conquest of Istanbul. This was such a huge event that according to some historians, the early modern period of history begins with this event. This is because, after the fall of Constantinople, the feudalism system in Europe ended.